tool/tsh/tsh.go

```diff
@@ -1794,15 +1794,24 @@ func printStatus(debug bool, p *client.ProfileStatus, isActive bool) {
 func onStatus(cf *CLIConf) error {
 	// Get the status of the active profile as well as the status
 	// of any other proxies the user is logged into.
+	//
+	// Return error if not logged in, no active profile, or expired.
 	profile, profiles, err := client.Status("", cf.Proxy)
 	if err != nil {
-		if trace.IsNotFound(err) {
-			fmt.Printf("Not logged in.\n")
-			return nil
-		}
```

Comment:
Let's keep the error overwrite here:
```
		if trace.IsNotFound(err) {
			return trace.WrapWithMessage(err, "Not logged in.")
		}
```

Reply:
Thank you! This results in this message on the terminal:

```
error: "Not logged in.\n\tnot logged in"
```

How best to improve the formatting?

Reply:
After merging `master`:

```
ERROR: Not logged in.
not logged in
```

Is the redundant message okay?

Reply:
Ahh, it already has that same message.
Then let's not special-case `trace.IsNotFound` and always return `trace.Wrap(err)`